app/assets/stylesheets/application.scss

```diff
@@ -41,10 +41,30 @@ nav {
   background-color: rgb(100, 100, 100);
   opacity: 0.3;
 }
+
+.flash_notice {
+  text-align: center;
+  color: green;
```

Comment:
Color `green` should be written in hexadecimal form as `#008000`<br>Color literals like `green` should only be used in variable declarations; they should be referred to via variable everywhere else.